clients/fides-js/src/lib/preferences.ts

```diff
@@ -58,9 +58,9 @@ async function savePreferencesApi(
 /**
  * Updates the user's consent preferences, going through the following steps:
  * 1. Update the cookie object based on new preferences
- * 2. Update the window.Fides object
- * 3. Save preferences to the `fides_consent` cookie in the browser
- * 4. Save preferences to Fides API or a custom function (`savePreferencesFn`)
+ * 2. Save preferences to the `fides_consent` cookie in the browser
+ * 3. Save preferences to Fides API or a custom function (`savePreferencesFn`)
+ * 4. Update the window.Fides object
```

Comment:
updating the window.Fides object needs to happen after the api call, otherwise it creates a race condition and potentially gets overwritten too soon from the `initialize` script that fires at the same time as this stuff is happening.